dali/pipeline/operators/decoder/nvjpeg_helper.h

```diff
@@ -113,6 +126,91 @@ void HostFallback(const uint8_t *data, int size, DALIImageType image_type, uint8
+      "\xA0\xF8\x86\xB3\x37\x89\x47\x12\xE6\xB5\xF5\xFB\xB4\x36\xD2\x96"
+      "\xEA\x90\x99\x84\x13\x55\x4A\xE3\x48\x67\x7D\xF2\x70\xED\xDA\x92"
+      "\xBD\xE9\xE0\x72\xFB\x8F\xFF\xC4\x00\x25\x11\x00\x01\x02\x05\x03"
+      "\x04\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x02\x11\x00\x03"
+      "\x04\x21\x31\x41\x51\x61\x14\x22\x71\x91\xA2\xB1\xF1\xFF\xDA\x00"
+      "\x08\x01\x02\x01\x01\x3F\x01\x4D\x24\xB0\x84\x3D\xD4\x12\x5D\xF1"
+      "\x93\x9F\x6E\x40\x6C\x34\x2A\x99\x33\xE8\xC9\x42\x5C\x07\xF5\xC6"
+      "\xD8\xBF\x9D\xE1\x0B\x1D\x24\xB4\x01\xDA\xCF\xCA\xAE\x7E\x23\xEC"
+      "\xF9\x89\xC2\x5D\x35\x38\x94\x95\x39\xD3\x80\x2F\x7F\xDD\xF4\x11"
+      "\xFF\xC4\x00\x21\x10\x00\x02\x01\x04\x02\x02\x03\x00\x00\x00\x00"
+      "\x00\x00\x00\x00\x00\x01\x02\x03\x00\x11\x12\x22\x13\x81\x41\x51"
+      "\x61\xA1\xF1\xFF\xDA\x00\x08\x01\x01\x00\x06\x3F\x02\x97\x8A\x25"
+      "\x58\x5D\x90\x29\x71\xB3\x59\x7C\x7A\xFC\xA9\xE1\x61\x16\x7C\x5B"
+      "\x3B\x00\x33\x93\xE0\x77\xF5\x52\x4D\x0C\x8A\x5D\x14\x62\x1C\xD8"
+      "\x26\xA2\xFD\xD6\xCD\x10\x58\xC6\x6D\x20\x7D\x4B\x7B\xBF\x55\xFF"
+      "\xC4\x00\x1C\x10\x01\x00\x01\x05\x01\x01\x00\x00\x00\x00\x00\x00"
+      "\x00\x00\x00\x00\x01\x11\x00\x21\x31\x41\x61\x51\x91\xFF\xDA\x00"
+      "\x08\x01\x01\x00\x01\x3F\x21\x38\x1B\x15\x22\x1F\x10\x4D\xF7\xA5"
+      "\x3D\x84\xC6\x18\x96\x2D\x46\x11\xC8\x50\x0E\xA0\xF4\x43\xB6\x4F"
+      "\xC0\x76\x92\xBC\x17\x8C\x2C\xDC\x27\x4B\x98\x54\x0A\xFF\xDA\x00"
+      "\x0C\x03\x01\x00\x02\x00\x03\x00\x00\x00\x10\xEF\xEF\xFF\xC4\x00"
+      "\x19\x11\x01\x01\x01\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00"
+      "\x00\x00\x01\x11\x21\x31\x00\x41\xFF\xDA\x00\x08\x01\x03\x01\x01"
+      "\x3F\x10\xBC\x28\x90\xB4\x33\x43\x1A\x11\xAF\x14\x81\x84\x5A\x85"
+      "\x0B\x82\x10\x8C\x80\x44\x28\x06\x96\xC3\x05\x60\x2D\x68\xE0\xC8"
+      "\x26\xCB\xB7\x72\x8B\x52\x68\x4E\x9D\x08\xBF\x42\x20\xD9\x03\xAF"
+      "\x7F\xFF\xC4\x00\x19\x11\x01\x01\x01\x01\x01\x01\x00\x00\x00\x00"
+      "\x00\x00\x00\x00\x00\x00\x01\x11\x21\x31\x00\x41\xFF\xDA\x00\x08"
+      "\x01\x02\x01\x01\x3F\x10\x9B\xC6\x90\x10\x46\x67\xE2\x29\x20\x4D"
+      "\x67\x93\x2A\xA1\x86\x00\x12\x08\x0E\x84\xAA\xAE\xA7\x58\x02\xA8"
+      "\x2D\x03\x46\x0A\xBC\xD8\x6C\x3E\x49\x1C\x43\x0A\x2D\xAA\x85\x42"
+      "\x81\x58\x04\x3D\xFF\xC4\x00\x18\x10\x01\x01\x01\x01\x01\x00\x00"
+      "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x11\x21\x00\x31\xFF\xDA"
+      "\x00\x08\x01\x01\x00\x01\x3F\x10\x4A\x4D\x7C\x09\x32\x21\x8A\xD2"
+      "\x8D\xE7\x12\xFE\x4B\x22\x5D\xC8\xAC\x08\x20\x41\x2F\x51\x3D\x5C"
+      "\x8D\x0E\x93\x80\xA4\x78\x88\xE3\x18\xD3\x32\x7A\x74\x6B\x60\x2D"
+      "\x54\x3F\xFF\xD9";
+
+  NVJPEG_CALL(nvjpegJpegStreamParse(handle, (unsigned char*)encoded_jpeg, sizeof(encoded_jpeg)-1, false, false,
+                                    state.jpeg_stream));
+  unsigned int C;
+  unsigned int widths[NVJPEG_MAX_COMPONENT];
+  unsigned int heights[NVJPEG_MAX_COMPONENT];
```

Comment:
Wouldn't `unsigned int` fail at linter? Just asking, no idea what will happen

Reply:
This code is mainly for testing